src/components/Table/Table.less

```diff
@@ -1,371 +1,299 @@
 @Table-row-height: @size-standard-height - 2;
```

Comment:
these less variables need to be cleaned up, there are some unused vars defined here.